src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenColonyHandler.java

```diff
@@ -161,6 +167,12 @@ public void onSyncDataUpdate(final EntityDataAccessor<?> data)
         }
     }
 
+    @Override
+    public boolean registered()
```

Comment:
what does this do?

Reply:
just an accessor for the value, used in debug info

Reply:
ah ok